src/sentry/templates/sentry/emails/transactions.html

```diff
@@ -36,6 +45,22 @@
             </td>
         </tr>
         {% endif %}
+        {% if transaction_duration and fcp %}
+        <tr>
+            <th>{% trans "FCP Delay" %}</th>
```

Comment:
Is it too long on the page to write this out rather than abbreviate it? Thinking of all the people like me who'd not immediately catch on to what this stands for.

Reply:
Good point, I'll raise this with the team, thanks!

Reply:
First Contentful Paint Delay is probs too long - clicking into this will take them to the issue right? If so, I don't think we need to worry bout it